Dockerfile.ocp

```diff
@@ -1,7 +1,9 @@
-FROM rhel7:latest
+FROM ubi8
 
-RUN yum update -y \
- && yum clean all
+RUN dnf update -y \
```

Comment:
we need to change dnf to yum everywhere

Reply:
> we need to change dnf to yum everywhere

Will update, at some stage during my testing the RHEL8 image I was using didn't have yum but it looks like the ubi8 image has it.